examples/3d/generate_custom_mesh.rs

```diff
@@ -0,0 +1,261 @@
+// ! This example demonstrates how to create a custom mesh,
+// ! assign a custom UV mapping for a custom texture,
+// ! and how to change the UV mapping at run-time.
+use bevy::prelude::*;
+use bevy::render::mesh::Indices;
+use bevy::render::mesh::VertexAttributeValues;
+use bevy::render::render_resource::PrimitiveTopology;
+
+// Define a "marker" component to mark the custom mesh. Marker componenets are often used in Bevy for
+// querying entities, even if they don't contain information within them.
```

Comment:
```suggestion
// Define a "marker" component to mark the custom mesh. Marker components are often used in Bevy for
// filtering entities in queries with With, they're usually not queried directly since they don't contain information within them.
```
typo + clarification about filtering